.github/workflows/cmake.yml

```diff
@@ -35,10 +35,14 @@ jobs:
     - name: "Install Ubuntu dependencies"
       # These packages are already part of the ubuntu-20.04 image:
       # cmake gcc-10 g++-10 shellcheck libgmp-dev
-      # These aren't:
+      # Packages below aren't.
+      #
+      # We add the cannonical-server/server-backports PPA to get updated QEMU releases without having to manage
```

Comment:
typo: "cannonical" -> "canonical"

Reply:
(in the commit message as well)

Reply:
Gah. One day I'll submit a PR without a typo.